project-base/storefront/cypress/integration/Tests/cart/productAddToCartFromProductVariant.cy.js

```diff
@@ -5,8 +5,7 @@ import { productClickFromPromotedProductsOnHomepage } from '../../Functions/home
 import { addProductVariantToCartFromProductDetail } from '../../Functions/productDetail';
 
 it('Product variant - Adding variant product to cart from product detail and check product in cart', () => {
-    cy.visit('/');
-    productClickFromPromotedProductsOnHomepage(products.philips32PFL4308.catnum, products.philips32PFL4308.name);
+    cy.visit(products.philips32PFL4308.url);
```

Comment:
Can it not be rewritten in the same way in other tests?

Reply:
Of course we can, but then we need to think about the scenarios itself. I am not sure if it is purpose for example testing links in the menu (clicks). If we decide to change the tests I would rather do it in separate task.